code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+		"If you need to drag multiple people either to safety or to space, bring a locker over and stuff them all in before hauling them off.",
+		"You can grab someone by clicking on them with the grab intent, then upgrade the grab by clicking on the grab button in your active hand. An aggressive grab will allow you to place someone on a table by clicking on it, or throw them by toggling on throwing.",
+		"Holding alt and left clicking a tile will allow you to see its contents in the top right window pane, which is much faster than right clicking.",
+		"The resist button will allow you to resist out of handcuffs, being buckled to a chair or bed, out of locked lockers and more. Whenever you're stuck, try resisting!",
+		"You can move an item out of the way by dragging it and then clicking on an adjacent tile with an empty hand.",
+		"Maintenance is full of equipment that is randomized every round. Look around and see if anything is worth using. There are many places around the ship to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!",
+		"As a Ghost, you can double click on people, bots, or the singularity to follow them.",
+		"Sometimes you won't be able to avoid dying no matter how good you are at the game. it's part of the game, so try not to stress too much about it.",
+		"When a round ends nearly everything about it is lost forever, leave your salt behind with it.",
+		"Sometimes a round will just be a bust. C'est la vie.",
+		"This is a game that is constantly being developed for. Expect things to be added, removed, fixed, and broken on a daily basis.",
+		"Sometimes admins will just do stuff. Roll with it.",
+		"Plenty of things that aren't traditionally considered weapons can still be used to slowly brutalize someone to death, get creative!",
+		"Toolboxes do more damage if they are full of things, but their contents will spill when you use them as a weapon.",
+		"Heads of Staff will be more willing to help you if you bring paperwork that's already filled out.",
+		"Chainswords and energy weapons can slice both walls and people.",
+		"You can use . (period) instead of : to speak into a radio.",
+		"The radio key .i will allow you to speak into a nearby intercom, .r will speak into a radio in your right hand, and .l will speak into your left. The microphone does not need to be enabled for this to work.",
+		"Your ID card's access determines what departmental channels you can set intercoms to.",
+		"You can fax papers between requests consoles by attacking the console with a paper. Make sure the paper tray is closed first!",
+		"No wall or window is 100% impervious to heat.",
+		"Never try to fight the hostile maintenance drones on your own.",
+		"There is no cloning, but you can respawn... as a different person, obviously.",
+		"The Newscasters are actively updated with Aurora lore... please read them.",
```

Comment:
Sort of true, though lore news on newscasters doesn't seem fully updated/easy to read?